docs/assets/js/search.js

```diff
@@ -112,4 +134,6 @@ if ( searchTerm ) {
 
   // Display the results of the search.
   displaySearchResults( results, searchStore );
```

Comment:
w/r/t L19

```suggestion
  if ( results.length > 0 ) {
    displaySearchResults( results, searchStore );
  }
```

Reply:
Refactored in https://github.com/cfpb/design-system/pull/479/commits/04489eac20472c1a1c293054c09e98dcee1c100b